### PR TITLE
Dark theme gradient

### PR DESCRIPTION
**Pull request by bhavneet2000**

### CODE REVIEW

File: client/src/components/DashboardComponent/sidemenu/Sidemenu.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ const Sidemenu = () => {
     <div>
       <div className="side">
         <Link to={"/dashboard"}>
-          <img src={Logo} alt="Logo" />
+          <img src={Logo_dark} alt="Logo" />
```
Comment (ankitmishra03) [inline client/src/components/DashboardComponent/sidemenu/Sidemenu.js:19]: change this according to theme

File: client/src/pages/index.css [hunk 2]
```diff
@@ -41,11 +43,11 @@
 
 .home{
   margin-top: 100px;
-  background-color: var(--bg-color);
 }
 
 .custom-navbar{
   background-color: var(--navbar-color);
+  background-image: var(--bg-image);
```
Comment (ankitmishra03) [inline client/src/pages/index.css:50]: where are we using this?

File: client/src/components/DashboardComponent/sidemenu/Sidemenu.js (hunk 1 above)
Comment (bhavneet2000) [reply client/src/components/DashboardComponent/sidemenu/Sidemenu.js:19]: Dashboard does't have dark theme.

File: client/src/pages/index.css (hunk 2 above)
Comment (bhavneet2000) [reply client/src/pages/index.css:50]: To change navbar color

**Review by ankitmishra03 — APPROVED** [review]

**Pull request merged**